chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+
+A summary of datasets on the HF Hub:
+
+|                | Train Hours | Domain                                | Speaking Style         | Transcription Casing | Transcription Punctuation | Recommended Usage                                 |
+|----------------|-------------|---------------------------------------|------------------------|----------------------|---------------------------|---------------------------------------------------|
+| LibriSpeech    | 960         | Audiobooks                            | Narrated               | ❌                    | ❌                         | Academic benchmarks                               |
+| Common Voice 9 | 2224        | Wikipedia text + crowd-sourced speech | Narrated               | ✅                    | ✅                         | Day-to-day speech                                 |
+| TEDLIUM        | 452         | TED talks                             | Narrated               | ❌                    | ❌                         | Technical scientific, political and social topics |
+| Voxpopuli      | 543         | European Parliament recordings        | Spontaneous            | ❌                    | ❌                         | Non-native English speakers                       |
+| GigaSpeech     | 10000       | Audiobook, podcast, youtube           | Narrated & spontaneous | ❌                    | ✅                         | Robustness over multiple domains                  |
+| SPGISpeech     | 5000        | Financial meetings                    | Narrated & spontaneous | ✅                    | ✅                         | Fully formatted transcriptions                    |
+
+
+Let’s take an example where we want to train a speech recognition system to transcribe speech on the topic of machine learning (highly relevant!). Our speech is going to be narrated (scripted), and we’re not worried about punctuation or casing on our text output. From our reference table, it looks like TED-LIUM is a good choice of dataset!
+
+[//]: # On review, this sub-section might be too involved. Happy to omit it and dive straight into how to load a dataset.
+
+### How to load an audio dataset
+Before we can handle any audio datasets, we need to make sure we have the right dependencies installed. Specifically, we'll need the "Audio" feature from datasets: https://huggingface.co/docs/datasets/installation#audio. This should take care of the main Python packages we need to read audio files from byte form and convert them to arrays.
+
+Right! Now we're ready to go ahead and download our data. For this tutorial, we'll use the [_LibriSpeech ASR_](https://huggingface.co/datasets/librispeech_asr) corpus. LibriSpeech is one of the most popular datasets for benchmarking speech recognition systems in both academia and industry. It consists of approximately 1000 hours of narrated audiobooks collected from the (LibriVox)[https://librivox.org/] project.
```

Comment:
I'm a bit confused here: above we say that we'll use the TEDLIUM dataset, but now we've switched to LibriSpeech. I personally like the TEDLIUM approach more, since LibriSpeech is too common IMO (we want to show something different to every other ASR tutorial :))

Reply:
I was going to go with TED-LIUM, but there very few models on the Hub trained on this dataset, and by extension no submissions to the leaderboard which is a bit of a shame! In contrast, LibriSpeech is very popular on the Hub and has a very active leaderboard with autoevaluate